docs/parameters.rst

```diff
@@ -2,3 +2,276 @@
 
 Parameters
 ==========
+
+OpenSCM follows a number of conventions related to parameters, we document these
+In OpenSCM a 'parameter' is anything which is passed to a model e.g. |CO2| emissions, equilibrium climate sensitivity, aerosol forcing scaling.
+
+Simple climate models come in many different shapes and forms hence we do not expect them to all be able to do everything.
+However, to be included in OpenSCM they must be able to understand all of OpenSCM's key parameters where here, 'understand' means that their adapters should either be able to use them or should throw sensible warning/error messages if they are used.
+
+Conventions
+-----------
+
+'Pre-industrial' refers to an unperturbed state of the climate.
+Individual adapters can translate this into whatever year they need to for their model, but they should do such translations with this definition in mind.
+
+'Reference period' refers to the period a given variable is reported relative to the mean of.
+For example, 'surface temperature relative to a 1961-1990 reference period' refers to surface temperatures relative to the mean of the period 1961-1990.
+Adapters can report variables without reference periods if these are not used internally by the models.
+However, if the model uses an internal reference period then this should be indicated in the reported variable names by appending ``(rel. to XXXX-YYYY)`` to the variable name.
+Hence, 'surface temperature relative to a 1961-1990 reference period' would become ``Surface Temperature (rel. to 1961-1990)`` [TODO: discuss with other authors how we want this to play out].
```

Comment:
One of the more difficult tasks for us ;) Let's have a chat about this issue in April

Reply:
any thoughts on this @swillner. This convention was the best I could come up with. Maybe there are better, machine readable ways?

Reply:
make TODO reference a live issue

Reply:
now references #167